app/forms.py

```diff
@@ -31,8 +33,12 @@ def __init__(self, *args, **kwargs):
         super(SocialLinkForm, self).__init__(*args, **kwargs)
         self.fields["content_type"].widget = HiddenInput()
         self.fields["object_id"].widget = HiddenInput()
-        self.fields["object_id"].disabled = True
         self.fields["content_type"].disabled = True
+        self.fields["object_id"].disabled = True
+
+        # These two fields are automatically "cleaned"/filled anyways
+        self.fields["content_type"].required = False
+        self.fields["object_id"].required = False
```

Comment:
As described in the commit, this is done to prevent errors regarding these fields being missing from popping up upon invalid data in the link content. Is there a better way of accomplishing this (e.g. would filtering out these errors be preferable)?